svelte/components/Nft/NftTransfer.svelte

```diff
@@ -37,7 +37,7 @@
 
 <div id="kredeum-transfer-nft">
   <div class="modal-content">
-    <a href="." title="Close" class="modal-close"><i class="fa fa-times" /></a>
+    <a href="./#" title="Close" class="modal-close"><i class="fa fa-times" /></a>
```

Comment:
prefer ` <a href="#" />`

Reply:
now we have a warning "# is not a valid href attribute" that is weird because i thought it was

Reply:
sorry my mistake, that's ` <a href="." />`

(that what is used everywhere else in the Dapp)

Reply:
yes but "." makes page refreshing, witch is not what we want to do at all
with "#" we have a warning but now refresh